runtime/executor/src/main/java/org/apache/nemo/runtime/executor/data/stores/GlusterFileStore.java

```diff
@@ -83,7 +83,7 @@ public Block createBlock(final String blockId) {
    * @throws BlockWriteException if fail to write.
    */
   @Override
-  public void writeBlock(final Block block) throws BlockWriteException {
```

Comment:
This SonarCloud issue was assigned to another team. Could you revert the codes changed in this file and resolve[ another issue](https://sonarcloud.io/project/issues?fileUuids=AWkltI3qgtFwgfOLu2Bs&id=org.apache.nemo%3Anemo-project&resolved=false&rules=squid%3ARedundantThrowsDeclarationCheck)?

Reply:
and you can refer to [this comment](https://github.com/apache/incubator-nemo/pull/258#discussion_r352971924) to resolve the issue.